core/src/test/java/org/testcontainers/containers/DockerComposeOverridesTest.java

```diff
@@ -34,39 +34,42 @@ public class DockerComposeOverridesTest {
 
     private static final int SERVICE_PORT = 3000;
 
-    private static final String SERVICE_NAME = "alpine_1";
+    private final String serviceName;
 
     private final boolean localMode;
 
     private final String expectedEnvVar;
 
     private final File[] composeFiles;
 
-    public DockerComposeOverridesTest(boolean localMode, String expectedEnvVar, File... composeFiles) {
+    public DockerComposeOverridesTest(
+        String serviceName,
+        boolean localMode,
+        String expectedEnvVar,
+        File... composeFiles
+    ) {
+        this.serviceName = serviceName;
         this.localMode = localMode;
         this.expectedEnvVar = expectedEnvVar;
         this.composeFiles = composeFiles;
     }
 
-    @Parameters(name = "{index}: local[{0}], composeFiles[{2}], expectedEnvVar[{1}]")
+    @Parameters(name = "{index}: serviceName[{0}] local[{1}], composeFiles[{3}], expectedEnvVar[{2}]")
     public static Iterable<Object[]> data() {
         return Arrays.asList(
             new Object[][] {
-                { true, BASE_ENV_VAR, new File[] { BASE_COMPOSE_FILE } },
-                { true, OVERRIDE_ENV_VAR, new File[] { BASE_COMPOSE_FILE, OVERRIDE_COMPOSE_FILE } },
-                { false, BASE_ENV_VAR, new File[] { BASE_COMPOSE_FILE } },
-                { false, OVERRIDE_ENV_VAR, new File[] { BASE_COMPOSE_FILE, OVERRIDE_COMPOSE_FILE } },
+                { "alpine-1", true, BASE_ENV_VAR, new File[] { BASE_COMPOSE_FILE } },
+                { "alpine-1", true, OVERRIDE_ENV_VAR, new File[] { BASE_COMPOSE_FILE, OVERRIDE_COMPOSE_FILE } },
+                { "alpine_1", false, BASE_ENV_VAR, new File[] { BASE_COMPOSE_FILE } },
+                { "alpine_1", false, OVERRIDE_ENV_VAR, new File[] { BASE_COMPOSE_FILE, OVERRIDE_COMPOSE_FILE } },
             }
         );
     }
 
     @Before
     public void setUp() {
         if (localMode) {
-            Assumptions
-                .assumeThat(LocalDockerCompose.executableExists())
-                .as("docker-compose executable exists")
-                .isTrue();
+            Assumptions.assumeThat(LocalDockerCompose.executableExists()).as("docker executable exists").isTrue();
```

Comment:
nit: I find the old description a bit more obvious. We check for `LocalDockerCompose.executableExists()` and the fact that it is `docker` with `compose` subcommand is an implementation detail :)

Reply:
In this class, we also don't know if it will be the `docker` executable or the `docker-compose` executable that will be used. What do we generally expect this class to run going forward, v1 or v2?

Reply:
> docker with compose subcommand is an implementation detail

right. 

> What do we generally expect this class to run going forward, v1 or v2?

We just set compose v2 by default everywhere so that's the expected one unless the env var or property is used disabling it.